exercises/Graphing-3-Python.md

```diff
@@ -10,16 +11,16 @@ look at the problem empirically for mammals.
 
 Download some
 [mammal life history data](http://esapubs.org/archive/ecol/E084/093/Mammal_lifehistories_v2.txt)
-from the web. You can do this either directly in the program using urllib or
+from the web. You can do this either directly in the program using `urllib` or
 download the file to your computer using your browser and import it from there.
 
 Import the data into a Pandas data frame. There are some extra blank lines at
 the end of this file, so get rid of them by using the optional `read_csv()`
-argument, `skip_footer=7`.
+argument, `skip_footer = 7`.
 
 Missing data in this file is specified by -999 and -999.00. Tell Pandas that
 these are null values using the optional `read_csv()` argument,
-`na_values=['-999', '-999.00']. This will stop them from being plotted.
+`na_values = ['-999', '-999.00']`. This will stop them from being plotted.
```

Comment:
Since this is a function argument there should be no spacing around =. See https://www.python.org/dev/peps/pep-0008/#other-recommendations